language/diem-framework/modules/0L/Testnet.move

```diff
@@ -42,13 +49,18 @@ module StagingNet {
     struct IsStagingNet has key { }
 
     public fun initialize(account: &signer) {
-        assert(Signer::address_of(account) == CoreAddresses::DIEM_ROOT_ADDRESS(), Errors::requires_role(190301));
+        assert(
+            Signer::address_of(account) == CoreAddresses::DIEM_ROOT_ADDRESS(),
+            Errors::requires_role(190301)
+        );
         move_to(account, IsStagingNet{})
     }
 
     public fun is_staging_net(): bool {
         exists<IsStagingNet>(CoreAddresses::DIEM_ROOT_ADDRESS())
     }
 
+    // Todo: no remove_staging_net()? Can this be a problem in production?
```

Comment:
@0o-de-lally FYI.

Reply:
StagingNet is only used for tests, it's not ever launched in prod.